lms/djangoapps/support/static/support/jsx/entitlements/components/Table/Table.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+import { Button, Hyperlink, Table } from '@edx/paragon';
+
+const entitlementColumns = [
+  {
+    label: 'User',
+    key: 'user',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Course uuid',
+    key: 'course_uuid',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Enrollment',
+    key: 'enrollment_course_run',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Mode',
+    key: 'mode',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Expired At',
+    key: 'expired_at',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Created',
+    key: 'created',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Modified',
+    key: 'modified',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Order',
+    key: 'order_number',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Actions',
+    key: 'button',
+    columnSortable: false,
+    hideHeader: false,
+    onSort: () => {},
+  },
+];
+
+const EntitlementTable = (props) => {
+  const entitlementData = props.entitlements.map((entitlement) => 
```

Comment:
Consider moving the mapping function to it's own method like `getEntitlementData` and then the component could look like

```javascript
const EntitlementTable = props => (
  <div>
    <Table data={getEntitlementData(props.entitlements)} columns={entitlementColumns} />
  </div>
)
```